### PR TITLE
:sparkles: Support for additional plugins / skip-publish

### PR DESCRIPTION
**Pull request by axdotl**
### Type of Change

- [ ] Bugfix
- [x] Enhancement / new feature
- [ ] Refactoring
- [ ] Documentation

### Description

- Support to define additional plugins
- Possibility to skip the publish step

### Checklist

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Review the [Contributing Guideline](../blob/main/CONTRIBUTING.md) and sign CLA
- [ ] Reference relevant issue(s) and close them after merging

### CODE REVIEW

**Review by kaitimmer — APPROVED** [review]

File: action.yml [hunk 1]
```diff
@@ -52,14 +59,23 @@ runs:
...
+          python -m pip install $p
+        done
 
     - name: Generate docs site
       shell: bash
       run: techdocs-cli generate --no-docker --verbose
 
     - name: Publish TechDocs site to AWS S3
-      if: inputs.publisher-type == 'awsS3' && inputs.aws-access-key-id != '' && inputs.aws-secret-access-key != '' && inputs.aws-region != ''
+      if: inputs.skip-publish == 'false' && inputs.publisher-type == 'awsS3' && inputs.aws-access-key-id != '' && inputs.aws-secret-access-key != '' && inputs.aws-region != ''
```
Comment (flaxel) [inline action.yml:78]: I would check with `true` because for now we could also write `blabla` and it would work. 😄

File: action.yml [hunk 2]
```diff
@@ -36,12 +36,19 @@ inputs:
   azure-account-key:
     description: 'Azure Account Name'
     required: false
+  additional-plugins:
+    description: 'List of additional python plugins to use for site creation'
+    required: false
+  skip-publish:
```
Comment (flaxel) [inline action.yml:42]: What is the benefit of this skip option? 🤔 Could we use the other values blank and it would also work?

Comment (axdotl) [reply action.yml:42]: That's true, but I like to have it explicit.

File: action.yml (hunk 1 above)
Comment (axdotl) [reply action.yml:78]: Had the same thought. But if you revert the statement to `skip-publish!='true'`, then this step is executed for value `blabla`. Both not perfect :)

**Review by flaxel — APPROVED** [review]

**Pull request merged**